### ISSUE TITLE
LD_PRELOAD support like libfns?

### ISSUE THREAD

**Issue by uli42** [sahlberg/libsmb2#220]
Is support for LD_PRELOAD like in your libnfs possible or planned?

**Comment by sahlberg** [comment]
It is possible but not really planned.  The one in libnfs was more like an fun toy/experiment.
It should not be very much work to add one based on the one in libnfs.

Do you want this type of LD_PRELOAD ?   I can add it but not until maybe next week.

**Comment by uli42** [comment]
No, I am not desperately needing it. I just found it a nice idea to have an
equivalent feature set for both libs.

**Comment by sahlberg** [comment]
Not really worth the effort.
You can't easily (or at all) replace openat() in modern glibc and pretty much everything depends on openat() in common userspace so :-(
